### ISSUE TITLE
genereted results

### ISSUE THREAD

**Issue by YIN95** [w86763777/pytorch-ddpm#4]
Hi, 
Thank you for the great work. 
I have an issue with the generation. This is the loss curve and 15k results. Not sure what's wrong, I just use the default config. 

![image](https://user-images.githubusercontent.com/35791079/172822313-7b52c150-cfb6-4a9f-9802-7a9f1e025476.png)
![image](https://user-images.githubusercontent.com/35791079/172822359-b30040c1-90b3-4372-b33a-499a0384a400.png)

**Comment by wjsun357** [comment]
> Hi, Thank you for the great work. I have an issue with the generation. This is the loss curve and 15k results. Not sure what's wrong, I just use the default config.
> 
> ![image](https://user-images.githubusercontent.com/35791079/172822313-7b52c150-cfb6-4a9f-9802-7a9f1e025476.png) ![image](https://user-images.githubusercontent.com/35791079/172822359-b30040c1-90b3-4372-b33a-499a0384a400.png)

I have the same issue. I wonder how you solved it.

**Comment by w86763777** [comment]
@wjsun357 can you create an issue an add the following information to let me reproduce the failure.
- pytorch version
- CUDA version

Maybe you can train it again to see whether this failure is a special case.
